schema_utils.py

```diff
@@ -265,3 +265,12 @@ def is_regex(obj):
     def matches_regex(obj, regex):
         """Ensures that `obj` (a string) matches the given regex."""
         raise NotImplementedError
+
+    @staticmethod
+    def is_valid_email_list(obj):
```

Comment:
May be tests for this?

Reply:
The test I added also checks this validator. Actually mostly tests the validator, Let me know if there is a better place to put this

Reply:
In that case move them to `schem_utils_test.py` file.

Reply:
I agree with @prasanna08.